templates/register.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Регистрация</title>
+</head>
+    <body>
+    <h1>Регистрация</h1>
+        <form action="{{ url_for('register') }}" method="post">
+            {{ form.csrf_token }}
+            <p>
+                {{ form.name.label }}<br>
+                {{ form.name(size=32) }}
+            </p>
+            <p>
+                {{ form.surname.label }}<br>
+                {{ form.surname(size=32) }}
+            </p>
+            <p>
+                {{ form.email.label }}<br>
+                {{ form.email(size=32) }}
+            </p>
+            <p>
+                {{ form.password.label }}<br>
+                {{ form.password(size=32) }}
+            </p>
+            <p>
+                <input type="submit" value="Зарегистрироваться">
+            </p>
```

Comment:
Соблюдаем принцип `DRY`
```suggestion
    {% for field in form if ield.name == "csrf_token" %}
    <p>{{ ield.label }}: {{ ield}}</p>
            {% for error in ield.errors %}
                <span style="color: red;">{{ error }}</span><br>
            {% endfor %}
    {% endif %}
    {% endfor %}
```
Причем строки
```html
            {% for error in ield.errors %}
                <span style="color: red;">{{ error }}</span><br>
            {% endfor %}
```
будут повторять flash сообщение об ошибке. Тут вы можете решить, что лучше: либо выводить сообщение тут, либо выше.
Кстати, строку 
```html
<input type="submit" value="Зарегистрироваться">
```
я тоже убрал, так как есть такое поле в форме.